web_scraper.py.py

```diff
@@ -0,0 +1,63 @@
+from bs4 import BeautifulSoup
+import requests
+
+def parse_pet(href):
+    html = requests.get(href)
+    html_text = html.text
+    soup = BeautifulSoup(html_text, 'lxml')
+    details = soup.find('ul', class_ = 'petdetails')
+    indents = details.find_all('li')
+    #W poniższych zmiennych zapisze dane, -1 oznacza brak danych
+    age = -1
+    breed = -1

```

Comment:
Do not use -1 for fields that are later refered to as strings. Possible alternatives are nulls or not setting them before refering to them and catching thrown exceptions